content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/count_products.rst

```diff
@@ -106,35 +123,47 @@ that the quantity on hand before the adjustment has not yet been updated to matc
 counted quantity.
 
 There are two ways to apply the new inventory adjustment. The first way is to click the
-:guilabel:`Apply` button on the line at the far right of the page. The second way is to click the
-checkbox on the far left of the line. Doing so reveals new button options at the top of the page,
-one of which is an :guilabel:`Apply` button. Clicking this button instead causes an
+:icon:`fa-save` :guilabel:`Apply` button on the line at the far right of the page. The second way is
+to click the checkbox on the far left of the line. Doing so reveals new button options at the top of
+the page, one of which is an :guilabel:`Apply` button. Clicking this button instead causes an
 :guilabel:`Inventory Adjustment Reference / Reason` pop-up window to appear.
 
+.. image:: count_products/apply-inventory-adjustment.png
+   :align: center
```

Comment:
Slightly getting ahead of myself here since this is something Sam catches in technical reviews, but we're no longer using the `align` tag for images. 

It's not required for this stage of review, but can you please remove all instances of `align` by the next review stage? Thank you :blush: 

Not sure if you already knew, but there's a find and replace shortcut in VSCode!
- CTRL + F
- Click the ">" arrow icon in the pop-up to reveal a drop-down menu
- Leave the 'Replace' field blank and click the 'Enter' key on your keyboard to find and remove that word

![image](https://github.com/user-attachments/assets/fb9d3fae-48d6-47e7-925d-c4bedb7b5270)

Reply:
Thank you!! I was going to delete them, but since Kara had left them (didn't realize this was her first PR until you'd pointed that out :) ), I was wondering if maybe this convention was supposed to be left for 17.0 docs. 

Thanks so much for this pro tip with the find + replace function in VSCode - so helpful! :D